src/contributor-guide/pull-request-tests.md

```diff
@@ -0,0 +1,131 @@
+---
+group: contributor-guide
+title: Pull Request Tests
+---
+
+The following topics are included in this guide:
+
+-  [Magento Automated Testing](#magento-automated-testing)
+-  [Running tests for a pull request](#run-pull-request-tests)
+-  [Related pull requests](#related-pull-requests)
+
+## Magento Automated Testing {#magento-automated-testing}
+
+Magento Automated Testing is a testing infrastructure that allows to run the set of all required Magento Automated Tests against code in Pull Requests. It can be triggered via a comment in a pull request. The running tests are represented both as Pull Request statuses and as Pull Requests checks.
+
+Below there's information about all kinds of tests available currently.
+
+### Semantic Version Checker
+
+Semantic Version Checker is a test that compares the changed code against the mainline and detects levels of code changes. It shows a list of changes grouped by the class (level) of changes (MAJOR, MINOR, PATCH) with explanation of the change (Class was removed, [public] Method implementation changed, [private] Property has been removed etc.) The allowed level is PATCH.
```

Comment:
This was definitely extended with MFTF SVC. Please talk to Alex Kolesnik

Reply:
Do you think we can skip adding the details and leave only the general description of the test for now?